recipes/unicorn/all/conandata.yml

```
@@ -2,7 +2,13 @@ sources:
   "1.0.3":
     url: "https://github.com/unicorn-engine/unicorn/archive/refs/tags/1.0.3.tar.gz"
     sha256: "64fba177dec64baf3f11c046fbb70e91483e029793ec6a3e43b028ef14dc0d65"
+  "2.0.0":
+    url: "https://github.com/unicorn-engine/unicorn/archive/refs/tags/2.0.0.tar.gz"
+    sha256: "67b445c760e2bbac663e8c8bc410e43311c7fc92df4dfa8d90e06a021d07f634"
 patches:
   "1.0.3":
     - patch_file: "patches/1.0.3-0001-cmake-crossbuild-support.patch"
       base_path: "source_subfolder"
+  "2.0.0.":
```

This is why the thooks haven't complained.
You weren't applying the patches
```suggestion
  "2.0.0":
```